src/java.desktop/windows/classes/sun/awt/windows/ThemeReader.java

```diff
@@ -124,12 +148,17 @@ private static Long getTheme(String widget) {
         }
 
         // mostly copied from the javadoc for ReentrantReadWriteLock
-        Long theme = widgetToTheme.get(widget);
+        Long theme = null;
+
+        if(dpiAwareWidgetToTheme.containsKey(dpi))
+        {
+            theme = dpiAwareWidgetToTheme.get(dpi).get(widget);
```

Comment:
Avoid using `contains` followed by `get`, use `get` instead and handle `null` value.